### PR TITLE
Sets store status keys to use EncodeUVariant.

### PR DESCRIPTION
**Pull request by BramGruneir**

### CODE REVIEW

File: storage/engine/keys.go [hunk 1]
```diff
@@ -53,7 +52,7 @@ func StoreStatKey(stat proto.Key) proto.Key {
 // StoreStatusKey returns the key for accessing the store status for the
 // specified store ID.
 func StoreStatusKey(storeID int32) proto.Key {
-	return MakeKey(KeyStatusStorePrefix, proto.Key(strconv.FormatInt(int64(storeID), 10)))
+	return MakeKey(KeyStatusStorePrefix, encoding.EncodeUvarint(nil, uint64(storeID)))
```
Comment (spencerkimball) [inline storage/engine/keys.go]: Don't you need to update the range_data_iter_test?

**Comment by spencerkimball** [comment]
LGTM

File: storage/engine/keys.go (hunk 1 above)
Comment (BramGruneir) [inline storage/engine/keys.go]: No, I removed the key from there because it only puts a status at the end of a scanner iteration.

**Comment by spencerkimball** [comment]
Cool, LGTM

**Pull request merged**